cmd/wardenkms/wardenkms.go

```diff
@@ -38,6 +39,27 @@ type Config struct {
 	HttpAddr string `env:"HTTP_ADDR, default=:8080"`
 
 	LogLevel slog.Level `env:"LOG_LEVEL, default=debug"`
+
+	ConsensusNodeThreshold uint `env:"CONSENSUS_NODE_THRESHOLD, default=1"`
+}
+
+type GrpcNodeConfig struct {
+	GRPCUrl      string
+	GRPCInsecure bool
+}
+
+type GrpcNodeConfigDecoder []GrpcNodeConfig
+
+func (sd *GrpcNodeConfigDecoder) Decode(value string) error {
```

Comment:
let's avoid having a pointer to a pointer (= slice)

```suggestion
func (sd GrpcNodeConfigDecoder) Decode(value string) error {
```

Reply:
Isn't it how the thing supposed to be?